econml/grf/_base_grf.py

```diff
@@ -1099,3 +1109,43 @@ def predict_projection_var(self, X, projector):
         X, projector = self._check_projector(X, projector)
         return self._predict_point_and_var(X, full=False, point=False, var=True,
                                            project=True, projector=projector)
+
+    def oob_predict(self, Xtrain):
+        """ Returns the relevant output predictions for each of the training data points, when
+        only trees where that data point was not used are incorporated.
+
+        Parameters
+        ----------
+        Xtrain : (n_training_samples, n_features) matrix
+            Must be the same exact X matrix that was passed to the forest at fit time.
+
+        Returns
+        -------
+        oob_preds : (n_training_samples, n_relevant_outputs) matrix
+            The out-of-bag predictions of the relevant output parameters for each of the training points
+        """
+
+        if self.warm_start_:
+            raise AttributeError("`oob_predict` is not available when "
+                                 "the estimator was fitted with `warm_start=True`")

```

Comment:
Should this be mentioned in the method docstring?  What about the warm_start part of the __init__ docstring?

Reply:
Added it in docstring. Also added it in the warm_strt docstring. I also removed warm_Start from causalForestDML. It makes no sense there as there is no way for the final model to be refit to new data! So that only is a weird option to have.